src/citadel/citadel.utils.ts

```diff
@@ -230,21 +235,65 @@ export async function getCitadelKnightingRoundsStats(): Promise<CitadelKnightsRo
   const client = new GraphQLClient(CITADEL_KNIGHTS_ROUND_SUBGRAPH_URL);
   const graphSdk = setKnightsRoundStatsSdk(client);
 
-  const knightsRoundStatResp = await graphSdk.KnightsRounds();
+  let totalVotes: VoteFragment[] = [];
+
+  let voteId;
+  while (true) {
+    const voteData = await graphSdk.Votes({
+      first: 2,
+      orderBy: Vote_OrderBy.Id,
+      orderDirection: OrderDirection.Asc,
+      where: {
+        id_gt: voteId,
+      },
+    });
+    const { votes } = voteData;
+    if (votes.length === 0) {
+      break;
+    }
+    totalVotes = totalVotes.concat(votes);
+  }
 
   const ctdlToken = await getFullToken(Chain.getChain(Network.Ethereum), TOKENS.CTDL);
+  const recordedVotes: Record<string, Record<string, number>> = {};
+
+  totalVotes.forEach((v) => {
+    const { knight, amount, voter } = v;
+    if (!recordedVotes[knight.id]) {
+      recordedVotes[knight.id] = {};
+    }
+    if (!recordedVotes[knight.id][voter.id]) {
+      recordedVotes[knight.id][voter.id] = 0;
+    }
+    recordedVotes[knight.id][voter.id] += formatBalance(amount, ctdlToken.decimals);
+  });
 
-  const totalWeight = knightsRoundStatResp.knights
-    .map((k) => Math.pow(k.voteAmount, 1 / 1.3))
+  const knightVoteWeights = Object.fromEntries(
+    Object.entries(recordedVotes).map((e) => {
+      const [knight, votes] = e;
+      const voteWeight = Object.values(votes).reduce(
+        (total, voteAmount) => (total += Math.pow(voteAmount, 1 / 1.3)),
+        0,
+      );
+      const voteCount = Object.values(votes).reduce((total, voteAmount) => (total += voteAmount), 0);
+      return [knight, { voteWeight, voteCount }];
+    }),
+  );
+
+  const totalWeight = Object.values(knightVoteWeights)
+    .map((i) => i.voteWeight)
     .reduce((total, weight) => (total += weight), 0);
 
-  return knightsRoundStatResp.knights.map((knight) => ({
-    knight: getKnightEnumByIx(Number(knight.id)),
-    votes: knight.voteCount,
-    voteWeight: (Math.pow(knight.voteAmount, 1 / 1.3) / totalWeight) * 100,
-    votersCount: knight.voters.length,
-    funding: formatBalance(knight.voteAmount, ctdlToken.decimals) * CITADEL_START_PRICE_USD,
-  }));
+  return Object.entries(knightVoteWeights).map((e) => {
+    const [knight, { voteCount, voteWeight }] = e;
+    return {
+      knight: getKnightEnumByIx(Number(knight)),
+      votes: voteCount,
```

Comment:
i believe here u have `voteAmount` in CTDL, not count of `votes`

Reply:
@mrbasado please confirm here, discussed offline but I think we do want the 'votes' to be the fully tallied amounts of CTDL purchased while the "vote Count" is the number of users that contributed those votes

Reply:
yeah, votes = total xctld, votercount = number of people, vote weight is thier share of supply (for the freefor all part)